resources/views/dashboard/_index/expiring.blade.php

```diff
@@ -0,0 +1,45 @@
+<div class="table-responsive <?php echo(! $expirings->isEmpty() ? "panel-scroll" : "") ?>">
```

Comment:
You get the idea 🤣

Reply:
Not a huge win. The project has now much bigger issues to solve and to spend time on.